src/app/models/hero.ts

```diff
@@ -0,0 +1,30 @@
+import { Equipment } from './equipment';
+
+export class Hero {
```

Comment:
why do you need this for the navbar?

Reply:
Navbar gets queryparams from mocked heroes hall when navigating by clicking on datafilled dummy heroes. At the moment only name and id is given. All stored in instantiated hero from hero model.